tests/tools/oecert/host/host.cpp

```diff
@@ -418,6 +430,26 @@ static oe_result_t _read_key(const char* filename, uint8_t** data, size_t* size)
 static oe_result_t _process_params(oe_enclave_t* enclave)
 {
     oe_result_t result = OE_FAILURE;
+    bool sgx_asem_env_set = false;
+
+// read "SGX_AESM_ADDR" environment variable
+// if not set, unset it back after oecert function
+#if defined(__linux__)
+    sgx_asem_env_set = (getenv(SGX_AESM_ADDR) != NULL);
```

Comment:
A high-level question: what happens if the out of proc is not supported by the environment? Is there a way to check such cases?

Reply:
`out-of-proc` depends on `sgx-quote-ex` library. If it's not installed, it will return sgx-quote-ex not found and quote generation will fail. Ryan's PR #3882 has added those dependencies in ansible script.

Reply:
What happens if users simply update the openenclave package without using ansible?

Reply:
Ah, I guess sgx-quote-ex should be one of the dependencies then. Is that correct?

Reply:
Yes, that's discussed at SIG-Attestation meeting. We will document this behavior so that user will know sgx-quote-ex is the dependencies for out-of-proc quoting.

Reply:
Miscommunicated. In the future given proper configuration, out-of-proc quote gen should work in Windows. Enforcing in-proc/out-of-proc should apply to both Linux and Windows. For now we only test out-of-proc quote gen in Linux.